### PR TITLE
add double quotes support to get_sp

### LINKED ISSUE #589: Double quotes support in get_sp
`get_sp('sp.key', default)` should also support `get_sp("sp.key", default)`

### PR DESCRIPTION
**Pull request by Bonczidai**
fixes #589 

Signed-off-by: Bonczidai Levente <levente.bonczidai@hpe.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
adding @SamMarkowitz - please check whther we need to update something in the docs

**Comment by SamMarkowitz** [comment]
the docs are fine. no need to add anything.

**Comment by Bonczidai** [comment]
thanks @SamMarkowitz  :wink:

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/utils/ExpressionUtils.java [hunk 1]
```diff
@@ -60,9 +64,10 @@ public static String extractExpression(Serializable value) {
     }
 
     public static Set<String> extractSystemProperties(String expression) {
-        Set<String> properties = matchFunction(SYSTEM_PROPERTY_PATTERN, expression, 1);
-        Set<String> propertiesWithDefault = matchFunction(SYSTEM_PROPERTY_PATTERN_WITH_DEFAULT, expression, 1);
-        properties.addAll(propertiesWithDefault);
+        Set<String> properties = matchFunction(SYSTEM_PROPERTY_PATTERN_SINGLE_QUOTE, expression, 1);
```
Comment (gituser173) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/utils/ExpressionUtils.java]: maybe rename matchFunction to getMatchedProperties or something similar WDYT?

**Comment by gituser173** [comment]
Looks good :+1:

**Pull request merged**